site/source/docs/tools_reference/emcc.rst

```diff
@@ -55,28 +55,33 @@ Options that are modified or new in *emcc* are listed below:
 .. _emcc-O2: 
 		
 ``-O2``
-	Like ``-O1``, but with various JavaScript-level optimizations and LLVM ``-O3`` optimizations. 
+	Like ``-O1``, but with various asm.js/wasm and JavaScript optimizations and LLVM ``-O3`` optimizations.
```

Comment:
I didn't realize this was true; I'm not sure it's a good idea because as I mentioned in the previous PR, O2 may well be better than O3 for the web use case :( Not sure it's worth changing out from under users though, since we recommend Os for releases anyway.

Reply:
If -O2 is better in some cases than both -O3 and -Os, we could consider running more non-LLVM opts (like the one in this PR) in -O2. Overall though I don't think we've seen a case that needs that.